docs/variants/dell_optiplex/building-manual.md

```diff
@@ -32,22 +41,100 @@ To build coreboot image, follow the steps below:
     docker run --rm -it \
        -v $PWD:/home/coreboot/coreboot \
        -w /home/coreboot/coreboot \
-       coreboot/coreboot-sdk:0ad5fbd48d /bin/bash
+       coreboot/coreboot-sdk:VERSION /bin/bash
     ```
 
+    * `VERSION` should be replaced according to version you building:
+      - `v0.1.0` - `2022-04-04_9a8d0a03db`
+
 5. Inside of the container, configure and start the build process:
 
     ```bash
-    (docker)make distclean
-    (docker)cp configs/config.dell_optiplex_9010_sff .config
-    (docker)make olddefconfig
-    (docker)make
+    make distclean
+    ```
+
+    ```bash
+    cp configs/config.dell_optiplex_9010_sff .config
+    ```
+
+    ```bash
+    make olddefconfig
+    ```
+
+    ```bash
+    make
     ```
 
     or simply:
 
     ```bash
-    (docker)make distclean && cp configs/config.dell_optiplex_9010_sff .config && make olddefconfig && make
+    make distclean && cp configs/config.dell_optiplex_9010_sff .config && make olddefconfig && make
+    ```
+
+This will produce a release binary placed in `build/coreboot.rom`. To flash
+coreboot refer to [initial deployment manual](initial-deployment.md).
+
+## Debug build
+
+1. Clone the coreboot repository:
```

Comment:
Do we really need the steps 1 and 2? Same with release build. As we have HTTPS links in the `.gitmodules` thiese steps should be obsolete.

Reply:
Yes, you are right. Fixed.

Reply:
@miczyg1 can you resolve that?